src/pages/Account/Tabs/TokensTab.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import {gql, useQuery} from "@apollo/client";
+import {TokensTable} from "../Components/TokensTable";
+
+const TOKENS_QUERY = gql`
```

Comment:
does this query give us all (currently) owned tokens? meaning it excludes ones they sold?

Reply:
It's a good question for @bowenyang007. My guess is that it excludes the sold ones.

Reply:
plz do amount > 0 lol